src/store/io/exportCX.ts

```diff
@@ -298,5 +291,7 @@ export const exportNetworkToCx2 = (
     ...aspects.map(({ key, aspect }) => ({ [key]: aspect })),
     { status },
   ]
+
+  console.log(cx)
```

Comment:
Need to remove this (can be a long console output)

Reply:
right, thanks